From f7dc9ff0c66474175c13be5ac63863686db855b5 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 23 Jun 2014 20:01:12 -0400
Subject: [scsi] qla2xxx: Update version number to 8.06.00.08.07.0-k3

Message-id: <1403553672-3844-7-git-send-email-cdupuis@redhat.com>
Patchwork-id: 84415
O-Subject: [PATCH RHEL 7.1 e-stor 6/6] qla2xxx: Update version number to 8.06.00.08.07.0-k3.
Bugzilla: 1090378
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

From: Chad Dupuis <chad.dupuis@qlogic.com>

RHEL 7 specific

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_version.h b/drivers/scsi/qla2xxx/qla_version.h
index 58fd647..75f546f 100644
--- a/drivers/scsi/qla2xxx/qla_version.h
+++ b/drivers/scsi/qla2xxx/qla_version.h
@@ -7,7 +7,7 @@
 /*
  * Driver version
  */
-#define QLA2XXX_VERSION      "8.06.00.08.07.0-k2"
+#define QLA2XXX_VERSION      "8.06.00.08.07.0-k3"
 
 #define QLA_DRIVER_MAJOR_VER 8
 #define QLA_DRIVER_MINOR_VER 6
-- 
1.7.1